### ISSUE TITLE
Generator missing on instantreset

### ISSUE THREAD

**Issue by rlf** [cybertiger/Bukkit-InstantReset#10]
Did the following:

1. Disabled `allow-nether` and `allow-end` in `bukkit.yml` and `server.properties`
2. Unloaded and deleted the worlds
3. Created new worlds using `/mv create ...`
4. Copied the worlds to `templates` folder
5. Configured IR - yes, did it the right way.
6. Tested IR, seemed to work fine
7. Teleported to a position far far away, created a nether portal, and went to the IR created nether world.
8. Ended up in a void-nether world (no chunks generated).
9. Unloaded both InstantReset and the world
10. Recreated it as per step 3.
11. Redid step 7. ended up in a wonderfully correctly generated nether world.

In other words - it seems the chunkgenerator fails when "importing" worlds using InstantReset.

It doesn't make sense, that IR only supports fully generated worlds - so perhaps I'm missing something here?

**Comment by cybertiger** [comment]
Feature was never implemented, for performance reasons any sane server admin should be using pregenerated worlds (to avoid having to regenerate them all after a reset and the performance degredation that would cause).

Right now it only supports a void chunk generator, this is intentional, please suggest a sane use case where you would not want pregenerated worlds.

**Comment by rlf** [comment]
Any server not having a world-border, or using plugins for managing world-generation (i.e. skyworld servers etc.).

I would call it a sane use case, since not all servers use pregenerated worlds. You might not.
